### DISCUSSION TITLE
Donations via liberapay to naveensingh are not possible without a PayPal account

### DISCUSSION THREAD

**Discussion by David-Else** [FossifyOrg/General-Discussion#110] [General]
It is impossible to donate without a PayPal account, and that is not happening, terrible company. If you add a Stripe account then you can enjoy more donations, thanks. https://liberapay.com/naveensingh

**Comment by naveensingh** [comment]
@David-Else 

Stripe still hasn't approved my application for Liberapay. In case you didn't know, you can use Github Sponsors for one-time donations too.

Also discussed here: https://github.com/FossifyOrg/General-Discussion/issues/33
